YARCH/Sources/Modules/Authorization/View/AuthLoginController.swift

```diff
@@ -0,0 +1,42 @@
+import UIKit
+
+final class AuthLoginController: UIViewController, AuthLoginView {
+
+    var onSignInButtonTap: (() -> Void)?
+    private var signInButton: UIButton = UIButton(type: .custom)
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
```

Comment:
View setup throught `loadView` is required. Please refer to `Catalog` and `CatalogDetails` modules. Please note `addSubviews`, `makeConstraints` methods and SnapKit layout setup as well.

Reply:
Ok.